cassandra-analytics-cdc-sidecar/src/test/java/org/apache/cassandra/cdc/CassandraClientSourceTest.java

```diff
@@ -64,8 +64,8 @@ public void testGetPrimaryKeyObjects()
         new TestValue("ks", "b", "text", ByteBuffer.wrap("test".getBytes(StandardCharsets.UTF_8)))
         );
         Object[] primaryKeyValues = CassandraClientSource.getPrimaryKeyObjects(TYPES, primaryKeyColumns);
-        assertEquals(1, primaryKeyValues[0]);
-        assertEquals("test", primaryKeyValues[1]);
+        assertThat(primaryKeyValues[0]).isEqualTo(1);
```

Comment:
"you" can also do this:
```suggestion
        assertThat(primaryKeyValues[0]).isOne();
```

Reply:
`isOne` is only available for numeric value assertions. `primaryKeyValues[0]` is `Object`.